package.json

```diff
@@ -86,6 +86,7 @@
     "prosemirror-tables": "^0.7.8",
     "prosemirror-utils": "^0.6.7",
     "react-popper": "^2.2.3",
+    "snarkdown": "^1.2.2",
```

Comment:
What a neat little parser!

Reply:
Isn't it? I love the package description: `basically one regex and a huge if statement`